NurVo_Frontend/src/pages/Bookmark.tsx

```diff
@@ -0,0 +1,132 @@
+import { useState, useEffect } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import axios from 'axios';
+
+import Colors from '../utilities/Color';
+import { Body011 } from '../utilities/Fonts';
+import { screenWidth } from '../utilities/Layout';
+import BookmarkList from '../components/BookmarkList';
```

Comment:
```suggestion
import BookmarkList from '../components/BookMarkList';
```
파일명이 달라서 에러가 발생합니다. 이런 부분은 pr 올리기 전 실행해보는 것으로 확인할 수 있으니 다른 파일에서 작업하시더라도 파일 합친 후 실행시켜서 확인해주세요